docs/authentication.md

```diff
@@ -65,6 +63,10 @@ auth()->id();
 user_id();
 ```
 
+> **Note**
+> The `auth_helper` is autoloaded by Composer. If you want to *override* the functions,
+> you need to define them in `app/Common.php`.
```

Comment:
I was wrong, Common.php does come first! That is good.

Reply:
> you need to define them in `app/Common.php`

This was unclear to me too!